pylint/checkers/base/comparison_checker.py

```diff
@@ -240,13 +257,15 @@ def _check_callable_comparison(self, node):
         "unidiomatic-typecheck",
         "literal-comparison",
         "comparison-with-itself",
+        "comparison-of-two-literals",
```

Comment:
```suggestion
        "comparison-of-constants",
```

Reply:
Okay, the lint has been renamed everywhere.